demos/text-generation/llm.js

```diff
@@ -20,13 +27,15 @@ export class LLM {
     session1 = undefined;
     session2 = undefined;
     feed = {};
+    fetches = {};
     outputTokens = [];
     stop = false;
     kvDims = [];
-    dataType = "float16";
     deviceType = "gpu";
     maxLength = 2048;
     mlContext = undefined;
+    startLen = 0;
```

Comment:
```suggestion
    startLength = 0;
```
Whole words strongly preferred.